### ISSUE TITLE
SpectrumPreference OnPreferenceChangeListener not working

### ISSUE THREAD

**Issue by ghost** [the-blue-alliance/spectrum#27]
SpectrumPreference OnPreferenceChangeListener not working

**Comment by rabross** [comment]
Could you please post some sample code?

**Comment by ghost** [comment]
@rabross 

```
final SpectrumPreference custom_color = (SpectrumPreference) findPreference("custom_color");

custom_color.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener() {
            @Override
            public boolean onPreferenceChange(Preference preference, Object o) {
                Log.e("dialog", "changed");
//This method not working, Log not showed
                return false;
            }
        });
```

**Comment by rabross** [comment]
@SweetDevelopers 
I agree, this doesn't work. I tested `OnPreferenceChangeListener` using [`SpectrumPreferenceCompat`](https://github.com/the-blue-alliance/spectrum/blob/master/spectrum/src/main/java/com/thebluealliance/spectrum/SpectrumPreferenceCompat.java) class and it works. You could try using that or wait for a fix?

@nwalters512 
The code in the repo does not seem to match the code coming in from maven. I guess a release is due?
The call to `callChangeListener()` is missing and including this should make `OnPreferenceChangeListener` work correctly. It is available in the latest revision of `SpectrumPreference`. Could you do a new release with this revision included?

**Comment by rabross** [comment]
Looks like I already fix this after the 0.5.0 release #24

**Comment by nwalters512** [comment]
@rabross you're exactly right, looks like my last release was way back on June 1st. I've been moving back to college these past few days so I hadn't had time to look at this. I'll do a release with your fix later today.

**Comment by nwalters512** [comment]
Fixed in [v0.6.0](https://github.com/the-blue-alliance/spectrum/releases/tag/v0.6.0)!
